src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -48,10 +48,13 @@ public class NpgsqlDatabaseModelFactory : DatabaseModelFactory
                 TimeSpan.FromMilliseconds(1000.0));
 
         /// <summary>
-        /// Tables which are considered to be system tables and should not get scaffolded, e.g. the support table
+        /// Tables and views which are considered to be system tables and should not get scaffolded, e.g. the support table
         /// created by the PostGIS extension.
         /// </summary>
-        [NotNull] [ItemNotNull] static readonly string[] SystemTables = { "spatial_ref_sys" };
+        [NotNull] [ItemNotNull] static readonly string[] SystemTablesAndViews =
```

Comment:
When does EF plan to switch to C# 8?

Reply:
Probably not in the 3.0 timeframe. It's not even sure whether BCL will be fully annotated by then...